test/sdk/charge_test.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Sdk.ChargeTest do
+  use ExUnit.Case
+
+  test "charge" do
+    request = %Ext.Sdk.Request{
+      payload: %{
+        email: "some@body.nice",
+        amount: "10000",
+        metadata: %{
+          custom_fields: [
+            %{value: "makurdi", display_name: "Donation for", variable_name: "donation_for"}
+          ]
+        },
+        bank: %{
+          code: "057",
+          account_number: "0000000000"
+        },
+        birthday: "1995-12-23"
+      }
+    }
+
+    {:ok, response} = Paystackex.Sdk.Client.charge(request)
```

Comment:
This is an example how to use unified SDK.
Sure, requirements for SDK can be different, but this approach allows us to minify time for adding new integration.
1. We just need to add config
2. Override some base client functions (in current case it’s headers)